charts/catalog/values.yaml

```diff
@@ -56,3 +56,6 @@ controllerManager:
   resyncInterval: 5m
   # Broker relist interval; format is a duration (`20m`, `1h`, etc)
   brokerRelistInterval: 24h
+  # Whether or not the controller supports a --broker-relist-interval flag. If this is 
+  # set to true, brokerRelistInterval will be used as the value for that flag
+  brokerRelistIntervalActivated: false
```

Comment:
@vaikas-google for every flag that is not backward-compatible (when `--broker-relist-interval` is passed to a `v0.0.3` controller-manager image, the process exits immediately), we should add a feature flag like this.

Regarding charts being versioned, they already are in the GH releases, but obviously not in the tip of `master`. However, the fact that we track versioned images in the tip of `master` means that we actually need this feature flag right now. What we are doing now is not a bad thing, it's just a growing pain.

In the future, I'd like to see the following additions to our release process:

1. We publish helm charts to a helm repo, just like we publish images to quay (iirc, quay now can host helm charts - I don't know any more details right now)
2. We change the image tags in our helm charts _back_ to `canary`
3. We modify the walk-through to point to the versioned helm charts in the helm repo